.gitattributes

```diff
@@ -0,0 +1,2 @@
+# Enable automatic end-of-line normalization
+* text=auto
```

Comment:
You apparently have Git's autocrlf setting enabled on your local install. I always disable the setting on mine. This .gitattributes file is a way to specify the setting on a per-repo level so everyone has the same settings regardless of their system default.

Without this, Git checks out all files with LF line ending on my machine, which causes some annoyances like Visual Studio changing the whole .csproj every time I add a new file to convert all the line endings to CR/LF.